portfolio/src/main/webapp/style.css

```diff
@@ -6,9 +6,26 @@
     font-family: "BarlowR";
```

Comment:
Using specialized fonts is good but specifying a fallback family such as `serif`, `sans-serif`, or `monospace` is typically useful in case your user doesn't have the font specified installed.